source/rust_verify/src/rust_to_vir_expr.rs

```diff
@@ -363,6 +385,39 @@ fn fn_call_to_vir<'tcx>(
         let header = Arc::new(HeaderExprX::Requires(Arc::new(vir_args)));
         return Ok(mk_expr(ExprX::Header(header)));
     }
+    if is_opens_invariants || is_opens_invariants_except {
+        return err_span_str(
+            expr.span,
+            "'is_opens_invariants' and 'is_opens_invariants_except' are not yet implemented",
+        );
+
+        /*
```

Comment:
If you're planning to merge this soon (which I recommend to avoid it diverging from `main`, which would make conflict resolution more complex), can you please remove this block comment?